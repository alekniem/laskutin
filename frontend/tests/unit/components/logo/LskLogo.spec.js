import LskLogo from "@/components/logo/LskLogo";
import { createWrapper } from "UT/lsk_test_helper";

describe("LskLogo", () => {
    it("should match snapshot", () => {
        const wrapper = createWrapper(LskLogo);
        expect(wrapper.html()).toMatchSnapshot();
    });
});
