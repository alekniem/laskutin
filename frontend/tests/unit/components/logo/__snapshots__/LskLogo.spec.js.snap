// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LskLogo should match snapshot 1`] = `
<svg width="150px"
     height="30px"
     viewbox="0 0 50 10"
     xmlns="http://www.w3.org/2000/svg"
     style="fill: #ffffff;"
>
  <path d="M.394 9.01h4.752V7.486H2.214V2.131H.394zM8.014 9.13c.656 0 1.28-.296 1.8-.677h.042l.127.55h1.481V6.061c0-1.736-.973-2.477-2.476-2.477-.89 0-1.715.254-2.52.699l.636 1.185c.635-.296 1.08-.444 1.524-.444.593 0 .952.169 1.016.55C7.23 5.7 6.3 6.314 6.3 7.5c0 .868.613 1.63 1.714 1.63zm.593-1.397c-.381 0-.572-.138-.572-.381 0-.36.318-.656 1.609-.698v.677c-.286.211-.614.402-1.037.402zM15.147 9.13c1.767 0 2.71-.762 2.71-1.757 0-.878-.466-1.355-2.054-1.714-.825-.19-1.1-.212-1.1-.445 0-.19.211-.296.55-.296.55 0 1.164.17 1.714.487l.805-1.08c-.561-.35-1.355-.74-2.35-.74-1.535 0-2.519.677-2.519 1.714 0 .825.53 1.376 1.99 1.693.889.19 1.164.233 1.164.487 0 .19-.19.317-.656.317-.677 0-1.408-.2-2.074-.592l-.762 1.122c.677.444 1.672.804 2.582.804zM19.105 9.01h1.82V7.719l.614-.678 1.164 1.969h1.969l-2.096-3.112 1.969-2.18h-1.969l-1.608 1.884h-.043V1.644h-1.82zM27.042 9.13c.74 0 1.27-.318 1.714-.826h.043l.127.699h1.482V3.711h-1.82V7.12c-.276.381-.488.466-.795.466-.476 0-.603-.254-.603-.847V3.711h-1.82v3.26c0 1.312.487 2.159 1.672 2.159zM35.213 9.13c.72 0 1.291-.148 1.757-.275l-.318-1.312c-.285.095-.603.169-.963.169-.677 0-1.026-.233-1.026-.984V5.13h2.095V3.71h-2.095V2.124H33.16l-.233 1.587-1.355.064V5.13h1.27v1.63c0 1.418.635 2.37 2.371 2.37zM40.271 9.01h1.82V3.718h-3.83v1.419h2.01zm.847-6.011c.677 0 1.164-.424 1.164-1.059S41.795.86 41.118.86s-1.165.445-1.165 1.08S40.44 3 41.118 3zM44.505 9.01h1.82V5.539c.275-.254.508-.402.847-.402.466 0 .614.254.614.846V9.01h1.82V5.75c0-1.312-.508-2.159-1.683-2.159-.772 0-1.344.36-1.767.762h-.042l-.127-.635h-1.482z">
  </path>
</svg>
`;
